src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2185,6 +2185,15 @@ void os::Linux::print_uptime_info(outputStream* st) {
   }
 }
 
+static void print_container_helper(outputStream* st, jlong j, const char* header) {
+  st->print("%s", header);
```

Comment:
Small nit: I' d probably add ": " here and remove those out of the caller arguments.

Reply:
True, this should be changed.
I also adjusted the parameter name of print_container_helper from header to metrics because header was a bit misleading.